src/reporting/tezos/TezosConseilClient.ts

```diff
@@ -90,7 +92,8 @@ export namespace TezosConseilClient {
     export async function getOperationGroup(serverInfo: ConseilServerInfo, network: string, operationGroupID: string): Promise<any[]> {
         const query = ConseilQueryBuilder.setLimit(ConseilQueryBuilder.addPredicate(ConseilQueryBuilder.blankQuery(), 'hash', ConseilOperator.EQ, [operationGroupID], false), 1);
 
-        return getTezosEntityData(serverInfo, network, OPERATION_GROUPS, query);
+        const r = await getTezosEntityData(serverInfo, network, OPERATION_GROUPS, query);
+        return r[0];
```

Comment:
We should validate that at least one result was returned, else return an explicit error.